src/os/bluestore/BlueStore.cc

```diff
@@ -7583,12 +7583,19 @@ void BlueStore::_txc_finish(TransContext *txc)
   }
 
   OpSequencerRef osr = txc->osr;
+  CollectionRef c;
+  bool empty;
   {
     std::lock_guard<std::mutex> l(osr->qlock);
     txc->state = TransContext::STATE_DONE;
+
+    empty = _osr_reap_done(osr.get(), &c);
```

Comment:
Since tehre is only one osr_reap_done caller we can just move it inline too and avoid the awkward * argument.  (I had to do somethign similar in wip-bluestore-rtc so i think we're headed in this direction anyway?)